src/app/_components/most-popular-section/tabs.tsx

```
@@ -0,0 +1,69 @@
+
+  // Fetch products for each of the top four occasions in parallel,
+// then merge each occasion object with its corresponding products array.
+  const productsByOccasion = await Promise.all(
+    topFour.map(async (occasion) => {
+      const products = await getProductsByOccasion(occasion._id);
+      return { ...occasion, products };
+    })
+  );
+
+  return (
+    <Tabs defaultValue={topFour[0]._id} className="w-full">
```

ESLint

[error] className value must use cn() function. Example: className={cn("...")}

(local-rules/require-cn-classname)